From 851ef9489eb62f2293f17b08218075dd3b54764f Mon Sep 17 00:00:00 2001
From: Stefan Kost <ensonic@users.sf.net>
Date: Tue, 10 May 2011 13:06:09 +0300
Subject: [PATCH 21/32] runxmlconf: update to latest testsuite version

---
 runxmlconf.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/runxmlconf.c b/runxmlconf.c
index 38b0ce4..8ffb0c9 100644
--- a/runxmlconf.c
+++ b/runxmlconf.c
@@ -516,7 +516,7 @@ static void
 xmlconfInfo(void) {
     fprintf(stderr, "  you need to fetch and extract the\n");
     fprintf(stderr, "  latest XML Conformance Test Suites\n");
-    fprintf(stderr, "  http://www.w3.org/XML/Test/xmlts20080205.tar.gz\n");
+    fprintf(stderr, "  http://www.w3.org/XML/Test/xmlts20080827.tar.gz\n");
     fprintf(stderr, "  see http://www.w3.org/XML/Test/ for informations\n");
 }
 
-- 
1.7.9.5

