From c31e06cc18ac5d876eb712dcd2d7e2cbe74832a6 Mon Sep 17 00:00:00 2001
From: Stefan Kost <ensonic@users.sf.net>
Date: Mon, 9 May 2011 12:52:28 +0300
Subject: [PATCH 24/32] python: flag two unused args

---
 python/libxml.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/libxml.c b/python/libxml.c
index b0043e6..cfdaac8 100644
--- a/python/libxml.c
+++ b/python/libxml.c
@@ -1745,7 +1745,7 @@ typedef struct
 typedef xmlValidCtxtPyCtxt *xmlValidCtxtPyCtxtPtr;
 
 static void
-libxml_xmlValidCtxtGenericErrorFuncHandler(void *ctx, int severity, char *str) 
+libxml_xmlValidCtxtGenericErrorFuncHandler(void *ctx, ATTRIBUTE_UNUSED int severity, char *str)
 {
     PyObject *list;
     PyObject *result;
@@ -1772,7 +1772,7 @@ libxml_xmlValidCtxtGenericErrorFuncHandler(void *ctx, int severity, char *str)
 }
 
 static void
-libxml_xmlValidCtxtGenericWarningFuncHandler(void *ctx, int severity, char *str) 
+libxml_xmlValidCtxtGenericWarningFuncHandler(void *ctx, ATTRIBUTE_UNUSED int severity, char *str)
 {
     PyObject *list;
     PyObject *result;
-- 
1.7.9.5

