From 2799d70e471c8ab02bfac6d4efe1149e3d5ba37a Mon Sep 17 00:00:00 2001
From: Peter Stuge <peter@stuge.se>
Date: Tue, 23 Oct 2012 09:41:31 +0200
Subject: [PATCH 27/28] configure.ac: Move VISIBILITY_CFLAGS into AM_CFLAGS

---
 configure.ac       |    3 +--
 libusb/Makefile.am |    2 +-
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 3f252bb..86b91ff 100644
--- a/configure.ac
+++ b/configure.ac
@@ -205,9 +205,8 @@ AM_CONDITIONAL([HAVE_SIGACTION], [test "x$have_sigaction" = "xyes"])
 AC_CHECK_HEADERS([sys/time.h])
 AC_CHECK_FUNCS(gettimeofday)
 
-AM_CFLAGS="-std=gnu99 -Wall -Wundef -Wunused -Wstrict-prototypes -Werror-implicit-function-declaration $nopointersign_cflags -Wshadow ${THREAD_CFLAGS}"
+AM_CFLAGS="-std=gnu99 -Wall -Wundef -Wunused -Wstrict-prototypes -Werror-implicit-function-declaration $nopointersign_cflags -Wshadow ${THREAD_CFLAGS} ${VISIBILITY_CFLAGS}"
 
-AC_SUBST(VISIBILITY_CFLAGS)
 AC_SUBST(AM_CFLAGS)
 AC_SUBST(LTLDFLAGS)
 
diff --git a/libusb/Makefile.am b/libusb/Makefile.am
index 12ada88..5a32467 100644
--- a/libusb/Makefile.am
+++ b/libusb/Makefile.am
@@ -37,7 +37,7 @@ else
 THREADS_SRC = os/threads_windows.h os/threads_windows.c
 endif
 
-libusb_1_0_la_CFLAGS = $(VISIBILITY_CFLAGS) $(AM_CFLAGS) \
+libusb_1_0_la_CFLAGS = $(AM_CFLAGS) \
 	-DLIBUSB_DESCRIBE=\"`git --git-dir "$(top_srcdir)/.git" describe --tags 2>/dev/null`\"
 libusb_1_0_la_LDFLAGS = $(LTLDFLAGS)
 libusb_1_0_la_SOURCES = libusbi.h core.c descriptor.c io.c sync.c $(OS_SRC) \
-- 
1.7.9.5

