From 5a988c5d45173a87143b76440980770c0216dc12 Mon Sep 17 00:00:00 2001
From: Guillermo Vidal <guillermo.vidal@continental-corporation.com>
Date: Sat, 19 Nov 2011 20:17:48 -0600
Subject: [PATCH 2/3] Added .gitignore

---
 .gitignore |    1 +
 1 file changed, 1 insertion(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..d15929d
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1 @@
+xdelta3
-- 
1.7.9.5

